src/frontend/js/widgets/Dashboard/utils/learnerRoutesPaths.tsx

```diff
@@ -42,6 +44,16 @@ export const LEARNER_DASHBOARD_ROUTE_LABELS = defineMessages<LearnerDashboardPat
     description: 'Label of the certificates view used in navigation components.',
     defaultMessage: 'My certificates',
   },
+  [LearnerDashboardPaths.ORDER_CERTIFICATES]: {
+    id: 'components.Dashboard.DashboardRoutes.certificates.order.label',
+    description: 'Label of the order certificates view used in navigation components.',
+    defaultMessage: 'My certificates',
+  },
+  [LearnerDashboardPaths.ENROLLMENT_CERTIFICATES]: {
+    id: 'components.Dashboard.DashboardRoutes.certificates.enrollment.label',
+    description: 'Label of the enrollment certificates view used in navigation components.',
+    defaultMessage: 'My enrollment certificates',
+  },
```

Comment:
IMO There should be only one entry the sidebar menu `My certificates`. Once on this view, the user should be able to switch between Certificates and Attestation of achievement.

Reply:
That's the case, the sidebar link to certificate with the default tab "certificate".
Here, we provide a link to be able to access this menu section on the certificate tab or on the attestation tab.